src/applications/mhv-secure-messaging/components/ReplyButton.jsx

```diff
@@ -21,6 +21,7 @@ const ReplyButton = props => {
     props.visible && (
       <button
         type="button"
+        id="reply-button"
```

Comment:
I suggest using `data-testid` for testing attributes

Reply:
its just to made it the same with other buttons (print, move and trash)

Reply:
I have a verification assertion with loop for the buttons. So the id  will be more usable in this case.....  Or I can use data-testid  as well but then I have to add it to all buttons (print, move, trash)